README.md

```diff
@@ -2,70 +2,80 @@
-* 📝 **[Signed packages](./docs/src/docs/next-steps.md):** ModelKits and their assets can be signed so you can be confident of their provenance.
+* 😻 **No GPU or internet:** KitOps doesn't require GPUs, internet connectivity, your email, or favorite limb. It's a free tool you can use anywhere.
+* 🤗 **Flexible:** ModelKits can be used with any AI, ML, or LLM project - even multi-modal models.
+* 🧰 **Data science + DevOps:** Simplify asset management and versioning for training, experimentation, integration, deployment, and operations.
+* 🏃‍♂️‍➡️ **Run locally:** Kit's Dev Mode lets your run an LLM locally, configure it, and prompt/chat with it instantly (coming soon).
 * 🐳 **Deploy containers:** Generate a Docker container as part of your `kit unpack` (coming soon).
 * 🚢 **Kubernetes-ready:** Generate a Kubernetes / KServe deployment config as part of your `kit unpack` (coming soon).
-* 🤗 **Flexible:** ModelKits can be used with any AI, ML, or LLM project - even multi-modal models.
+* 📝 **Signed packages:** ModelKits and their assets can be signed so you can be confident of their provenance.
 
 ### See KitOps in Action
 
 https://github.com/jozu-ai/kitops/assets/4766570/05ae1362-afd3-4e78-bfce-e982c17f8df2
 
 ### What is in the box?
 
-**[ModelKit](./docs/src/docs/modelkit/intro.md):** At the heart of KitOps is the ModelKit, an OCI-compliant packaging format for sharing all AI project artifacts: datasets, code, configurations, and models. By standardizing the way these components are packaged, versioned, and shared, ModelKits facilitate a more streamlined and collaborative development process that is compatible with any MLOps or DevOps tool.
+**ModelKit:** At the heart of KitOps is the ModelKit, an OCI-compliant packaging format for sharing the artifacts involved in the AI/ML model lifecycle: datasets, code, configurations, and models. By standardizing the way these components are packaged, versioned, and shared, ModelKits facilitate a more streamlined and collaborative development process that is compatible with nearly any tool.
+
+**Kitfile:** A ModelKit is defined by a Kitfile - your AI/ML project's blueprint. It uses YAML to describe where to find each of the artifacts that will be packaged into the ModelKit along with metadata about each of them. Reading the Kitfile gives you a quick understanding of what's involved in each AI project.
+
+**Kit CLI:** Your magic wand for AI/ML collaboration. The Kit CLI not only enables users to create, manage, run, and deploy ModelKits -- it lets you pull only the pieces you need. Just need the serialized model for deployment? Use `unpack --model`, or maybe you just want the training datasets? `unpack --datasets`. Whether you are packaging a new model for development or deploying an existing model into production, the Kit CLI provides the flexibility and power to streamline your workflow.
```

Comment:
These doesn't have the link to the docs anymore, is that wanted or a mistake?

Reply:
That is a mistake